From c3ca8c66afd84c9a300261bd2671d7ce43624ef5 Mon Sep 17 00:00:00 2001
From: Bastian Blank <bastian.blank@credativ.de>
Date: Thu, 6 Mar 2014 13:43:14 +0100
Subject: Update defaults

Set default socket to unix:/var/run/milter-regex/sock and umask to 07.
Document in man-page.
---
 milter-regex.8 | 2 +-
 milter-regex.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/milter-regex.8 b/milter-regex.8
index 065caec..a4bbcc1 100644
--- a/milter-regex.8
+++ b/milter-regex.8
@@ -70,7 +70,7 @@ Ignore mail body after the specified number of lines.
 .It Fl p Ar pipe
 Use the specified pipe to interface
 .Xr sendmail 8 .
-Default is unix:/var/spool/milter-regex/sock.
+Default is unix:/var/run/milter-regex/sock.
 .It Fl u Ar user
 Run as the specified user instead of the default, _milter-regex.
 When
diff --git a/milter-regex.c b/milter-regex.c
index 0281748..037613d 100644
--- a/milter-regex.c
+++ b/milter-regex.c
@@ -96,7 +96,7 @@ static void		 usage(const char *);
 static void		 msg(int, struct context *, const char *, ...);
 
 #define USER		"_milter-regex"
-#define OCONN		"unix:/var/spool/milter-regex/sock"
+#define OCONN		"unix:/var/run/milter-regex/sock"
 #define RCODE_REJECT	"554"
 #define RCODE_TEMPFAIL	"451"
 #define XCODE_REJECT	"5.7.1"
@@ -782,7 +782,7 @@ main(int argc, char **argv)
 		perror("daemon");
 		goto done;
 	}
-	umask(0177);
+	umask(07);
 
 	msg(LOG_INFO, NULL, "started: %s", rcsid);
 	r = smfi_main();
